equinix/resource_fabric_cloud_router_sweeper.go

```diff
@@ -0,0 +1,69 @@
+package equinix
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/equinix/equinix-sdk-go/services/fabricv4"
+	equinix_errors "github.com/equinix/terraform-provider-equinix/internal/errors"
+	"github.com/equinix/terraform-provider-equinix/internal/sweep"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"log"
+)
+
+func AddTestSweeper() {
+	resource.AddTestSweepers("equinix_fabric_cloud_router", &resource.Sweeper{
+		Name:         "equinix_fabric_cloud_router",
+		Dependencies: []string{},
+		F:            testSweepCloudRouters,
+	})
+}
+
+func testSweepCloudRouters(region string) error {
+	var errs []error
+	log.Printf("[DEBUG] Sweeping Fabric Cloud Routers")
+	ctx := context.Background()
+	meta, err := sweep.GetConfigForFabric()
+	if err != nil {
+		return fmt.Errorf("error getting configuration for sweeping Fabric Cloud Routers: %s", err)
+	}
+	meta.Load(ctx)
```

Comment:
There is a linting failure on this line that will need to be addressed.  To satisfy the linter you'll need to change this to `err = meta.Load(ctx)` and then do something if the error is not nil.